webalyzer/collector/static/collect.js

```diff
@@ -67,6 +64,7 @@
 
   function postHTML(html, callback) {
     var data = new FormData();
+    html = options.beforeSendHTML(html, document.location.href);
```

Comment:
When you do the merge make sure this line ends up as the first thing in postHTML (ie before the source_hash check.)